x-pack/legacy/plugins/siem/server/lib/detection_engine/types.ts

```diff
@@ -4,6 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
```

Comment:
curious - should this import use 'kibana/server' instead of the relative import? I remember seeing some sort of message saying when to use one over the other but not sure what situation warrants what.